<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>1080</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>1080</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <widget class="QGroupBox" name="mainPage_groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>271</width>
          <height>1080</height>
         </rect>
        </property>
        <property name="title">
         <string>Main Menu</string>
        </property>
        <widget class="QPushButton" name="lockToProgram">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>231</width>
           <height>61</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Lock the computer to this Program</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>lock.png</normaloff>lock.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="SetNotification">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>220</y>
           <width>231</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>Set Notification time on this program</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>setting.svg</normaloff>setting.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
        <widget class="QCommandLinkButton" name="exitButton">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>10</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="CurrentProgram">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>4</number>
         </property>
         <property name="midLineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string>Current Program</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
        </widget>
        <widget class="QLabel" name="ProgramName">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="lineWidth">
          <number>4</number>
         </property>
         <property name="midLineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string>xProgramName</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>190</y>
           <width>47</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Or</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QWidget" name="LockedPage" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>110</y>
           <width>275</width>
           <height>800</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>275</width>
           <height>500</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <widget class="QLabel" name="locked">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>271</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="lineWidth">
           <number>4</number>
          </property>
          <property name="midLineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string>Your computer is successfully locked</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
         <widget class="QLabel" name="onlyOnThose">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>271</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="lineWidth">
           <number>4</number>
          </property>
          <property name="midLineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string>You can now only work on those programs</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
         <widget class="QPushButton" name="AddMoreButton">
          <property name="geometry">
           <rect>
            <x>174</x>
            <y>330</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Add more</string>
          </property>
         </widget>
         <widget class="QPushButton" name="unlockButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>370</y>
            <width>231</width>
            <height>61</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Unlock</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>unlock.png</normaloff>unlock.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
         </widget>
         <widget class="QLineEdit" name="notificationTime">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>330</y>
            <width>113</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QListWidget" name="ListPrograms">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>256</width>
            <height>192</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <widget class="QGroupBox" name="settingsPage_groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>271</width>
          <height>400</height>
         </rect>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <widget class="QLabel" name="NotificationStrength">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>271</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Notification Strength</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>231</width>
           <height>22</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="NotificationType">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>80</y>
           <width>271</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Notification Type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>231</width>
           <height>181</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="back_page_Button">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>310</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
